wagtailmedia/admin_urls.py

```diff
@@ -10,5 +10,6 @@
 
     url(r'^chooser/$', chooser.chooser, name='chooser'),
     url(r'^chooser/(\d+)/$', chooser.media_chosen, name='media_chosen'),
+    url(r'^(\w+)/chooser/upload/$', chooser.chooser_upload, name='chooser_upload'),
```

Comment:
I’m not sure I understand the `(\w+)`  at the start of the URL. is that the media type?

Reply:
Yes, it would be audio or video. I think it's ignored on backend code it's just for frontend UI tabs